connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/model/OasAdapter.java

```diff
@@ -0,0 +1,24 @@
+package org.citrusframework.openapi.model;
+
+import io.apicurio.datamodels.core.models.Node;
+
+public class OasAdapter<S extends Node, T> {
```

Comment:
this is a record as well. I've updated it:

```java
public record OasAdapter<S extends Node, T>(S node, T adapted) {

}
```

or do you intend to overwrite it somewhere?